resources/qml/TimelineSectionHeader.qml

```diff
@@ -92,8 +92,8 @@ Column {
                 height: fontMetrics.ascent
                 width: height
 
-                sourceSize.width: fontMetrics.lineSpacing
-                sourceSize.height: fontMetrics.lineSpacing
+                sourceSize.width: fontMetrics.ascent
+                sourceSize.height: fontMetrics.ascent
```

Comment:
You should probably just set this to height and width (also in the other places), but I guess this is fine for now.

Reply:
Set those to height/width in various places, haven't found a binding loop after some testing.